eclipse-mosquitto/content.md

```diff
@@ -0,0 +1,42 @@
+# What is Eclipse Mosquitto?
+
+Eclipse Mosquitto is an open source implementation of a server for version 3.1 and 3.1.1 of the MQTT protocol. Main homepage: http://mosquitto.org/
+
+%%LOGO%%
+
+# How to use this image
+
+##Mount Points
+
+Three mount points have been created in the image to be used for configuration, persistent storage and logs.
+
+	/mosquitto/config
+	/mosquitto/data
+	/mosquitto/log
+
+##Configuration
+
+When running the image, the default configuration values are used. To use a custom configuration file, mount a **local** configuration file to `/mosquitto/conf/mosquitto.conf`
+
+	docker run -it -p 1883:1883 -p 9001:9001 -v mosquitto.conf:/mosquitto/data/mosquitto.conf eclipse-mosquitto
+
+Configuration can be changed to:
+
+-	persist data to `/mosquitto/data`
+-	log to `/mosquitto/log/mosquitto.log`
+
+i.e. add the following to `mosquitto.conf`:
+
+	persistence true
+	persistence_location /mosquitto/data/
+	log_dest file /mosquitto/log/mosquitto.log
+
+**Note**: If a volume is used, the data will persist between containers.
+
+##Run
+
+Run a container using the new image:
+
+	docker run -it -p 1883:1883 -p 9001:9001 -v mosquitto.conf:/mosquitto/data/mosquitto.conf -v /mosquitto/data -v /mosquitto/log eclipse-mosquitto
+
+:boom: if the mosquitto configuration (mosquitto.conf) was modified to use non-default ports, the docker run command will need to be updated to expose the ports that have been configured.
```

Comment:
Heh, this `:boom:` likely renders in GitHub, but won't render on Docker Hub, unfortunately. :disappointed: